outreachy_proposal.md

```diff
@@ -0,0 +1,198 @@
+<hgroup>
+
+# Outreachy Proposal:
+
+## Crowdsourcing Translation for CHAOSS Diversity & Inclusion Badging
+
+</hgroup>
+
+> Ore-Aruwaji Oloruntola
+> Landmark University Omu-aran
+> Kwara, Nigeria
```

Comment:
```suggestion
> Ore-Aruwaji Oloruntola
>
> Landmark University Omu-aran
>
> Kwara, Nigeria
```

- Safer to always use 2 line-breaks, apparently GitHub will switch between 1 and 2 line-breaks in `<blockquotes>` depending on where they are used (see [Rich Diff](https://github.com/thecraftman/my_proposal/pull/1/files?short_path=c01bfc2#diff-c01bfc27ae111ed2225ba26c710fba39)).